### PR TITLE
Add mock for other row in table

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket

https://trello.com/c/JHKHOrzX/294-feature-budget-summary-navigation

# What solved
- [X] Add mock row in case that are more than 16 rows count category and subcategory
- [X] Should have a dark mode
- [X] Should have the table show (Others) rows, when there is more subcategory to show, and this should be a resume of the rest

# Description
This order the tables for the category with more subcategory
Add dark mode
 Add mock for the row others when there are more than 16 rows count all the tables and their category

### CODE REVIEW

File: src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx [hunk 1]
```diff
@@ -29,6 +29,10 @@ const FinancesTable: React.FC<Props> = ({ className, breakdownTable, metrics, pe
   const showQuarterly = !isMobile && period === 'Quarterly';
   const showMonthly = desk1440 && period === 'Monthly';
   const arrayMetrics = new Array<number>(iteration).fill(0);
+  const showFooter = true;
+  // Show color for others depending if odd or even
+  const isPar = orderData[`${tables[tables.length - 1]}`]?.length % 2 === 0;
```
Comment (YasielCabrera) [inline src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx:34]: could we rename this var to keep it in English :)

File: src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx [hunk 2]
```diff
@@ -63,6 +67,18 @@ const FinancesTable: React.FC<Props> = ({ className, breakdownTable, metrics, pe
               </TableRow>
             ))}
           </TableBody>
+          {index === tables.length - 1 && showFooter && (
+            <Footer isLight={isLight} isPar={isPar}>
```
Comment (YasielCabrera) [inline src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx:71]: there is a `isPar` here too

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx:34]: you are right, done

File: src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx (hunk 2 above)
Comment (alejandrocabriales) [reply src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx]: yes, its the same variable than is pass to component, its done

**Review by YasielCabrera — APPROVED** [review]
lgtm

**Pull request merged**